daemon/src/seed.rs

```diff
@@ -157,3 +157,30 @@ impl From<[u8; 32]> for UmbrelSeed {
         Self(bytes)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    pub struct PlaceholderSeed();
+    impl Seed for PlaceholderSeed {
+        fn seed(&self) -> Vec<u8> {
+            [0u8; 256].to_vec()
+        }
+    }
+
+    #[test]
+    fn given_placeholder_seed_then_placeholder_peer_id() {
+        // This test showcases that the placeholder peer-id is derived from a seed filled with
+        // Zeros.
+
+        let placeholder_seed = PlaceholderSeed();
+        let placeholder_peer_id = placeholder_seed.derive_identities().peer_id();
+
+        let expected_placeholder = "12D3KooWE1krEN7utXUFhLmLXLWPSuah5WQ7ChzUFetkMS7TRXsm"
+            .parse::<PeerId>()
+            .unwrap();
+
+        assert_eq!(expected_placeholder, placeholder_peer_id)
+    }
```

Comment:
I don't really understand the need for this test. 
Maybe you can add a constant and add a doc-test to show how you derived the constant. That way you can use the constant in different places. You can also annotate the constant with `cfg(test)` if it is only needed for tests.

Reply:
We don't need this test, but I wanted to showcase where the constant comes from. I will remove it and document it as part of the const.
The problem is that in `model` I can't access the seed from `daemon` unless I add a dependency. I don't think we should add cross-dependencies between these crates.